deploy/velero-configuration/hive-specific/111-velero.Schedules.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: velero.io/v1
+kind: Schedule
+metadata:
+  name: 5min-full-backup
+  namespace: openshift-velero
+spec:
+  schedule: '*/5 * * * *'
+  template:
+    includedNamespaces:
+    - '*'
+    ttl: 0h25m0s
```

We need this so we don't snapshot drives, just objects in etcd:
```suggestion
    snapshotVolumes: false
    ttl: 0h25m0s
```